### DISCUSSION TITLE
Add a CSS variable to justify text

### DISCUSSION THREAD

**Discussion by mscheltienne** [pradyunsg/furo#676] [Ideas] [answered]
Unless I missed it, I don't find a simple way to justify the text. For now, I added `html_css_files = ["css/justify.css"]` in `conf.py` and `_static/css/justify.css` which contains:

```
section {
    text-align: justify;
}
```

I am not sure if this is the best way to do it, or if Furo even has an existing variable that can be set to achieve this behavior. If not, it would be a nice to have feature.

**Comment by pradyunsg** [comment, marked as answer]
> Unless I missed it, I don't find a simple way to justify the text.

There is, and you found it! 😉 

More seriously, I can't really add a knob for every potential style of customisation that a user might want to do. To that end, I won't be adding a variable or some configuration value to do this, since this is fairly niche IMO and already something that one can achieve with existing extensibility options.

If more folks express interest for this (drop a reply to this!), I'm happy to reconsider this. :)
